### ISSUE TITLE
Has anyone compared to SAM6D?

### ISSUE THREAD

**Issue by GhYang0519** [NVlabs/FoundationPose#82]
I've tried comparing some of the more heavily occluded scenes, and SAM6D seems to work better than FoundationPose
Unfortunately, my computer broke down and I didn't save the result image...
Has anyone made some comparisons?
By the way, I only have a textureless mesh of objects, will this affect the effect?

**Comment by GhYang0519** [comment]
@wenbowen123

**Comment by wenbowen123** [comment]
>textureless mesh 

if the object has texture in reality, but your mesh does not have texture, that would certainly affect the results, cause FoundationPose will also leverage RGB cues.

**Comment by GhYang0519** [comment]
Oh, I understand, but SAM6D works well in the same situation. I wonder if I haven't set it up correctly regarding FoundationPose?
Can you explain the proportion of texture information's impact on FoundationPose?
Later (after I fix my computer), I will display some comparison images of the effects on my custom dataset.

**Comment by wenbowen123** [comment]
See this [example](https://private-user-images.githubusercontent.com/23078192/317560155-9b5bedde-755b-44ed-a973-45ec85a10bbe.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MjI0MzIsIm5iZiI6MTcxMzkyMjEzMiwicGF0aCI6Ii8yMzA3ODE5Mi8zMTc1NjAxNTUtOWI1YmVkZGUtNzU1Yi00NGVkLWE5NzMtNDVlYzg1YTEwYmJlLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAxMjg1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE2NGMyMTFmYjc4YmQ3NzU3MTllMjYxZjc1YjUyNjk2ZjJkNzVkOTA0YTlhNDMxZGRjZDAyMzEyNzdlZWEzYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WUXgDQXzMPyoMPUGISvBgwSB6kbOhZ0hYLfJOp2bWs0) how FoundationPose distinguishes the rotation of the tomato can by using the RGB cues. If the mesh's color deviate a lot from the scene (e.g. untextured mesh), it will cause a big performance drop.

What objects are you using and what prevents you of getting the texture?

**Comment by GhYang0519** [comment]
Thank you for your reply.
Yes, I have looked at the example and the effect is great. It seems that RGB information is still quite important in FoundationPose.
Regarding my Mesh, I used a 3D camera to capture my object point cloud and GT Pose from multiple perspectives, then concatenated them and obtained them using o3d.create_from_point_cloud_poisson. Regarding normal, I used o3d.compute_triangle_mormals and o3d.compute_vertex_mormals.
